spm-monitor-storm/pom.xml

```diff
@@ -96,6 +91,48 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-shade-plugin</artifactId>
         <version>2.3</version>
+        <executions>
+          <execution>
+            <id>package-shaded-implementation-jar</id>
+            <phase>package</phase>
+            <goals>
+              <goal>shade</goal>
+            </goals>
+            <configuration>
+              <transformers>
+                <transformer
+                        implementation="org.apache.maven.plugins.shade.resource.ManifestResourceTransformer">
+                  <manifestEntries>
+                    <Specification-Version>${project.version}</Specification-Version>
+                  </manifestEntries>
+                </transformer>
+              </transformers>
+              <artifactSet>
+                <excludes>
+                  <exclude>com.sematext.spm:spm-monitor-utils</exclude>
+                  <exclude>com.sematext.spm:spm-tracing-agent</exclude>
+                  <exclude>com.sematext.spm:spm-tracing-agent-impl</exclude>
+                  <exclude>com.sematext.spm:spm-monitor-starter</exclude>
```

Comment:
Agent works well without all these excluded modules?

Reply:
Yes. I verified by sending Strom metrics to an app in apps.sematext.com

Reply:
Following modules are already excluded from deps jar in parent pom

```
 <exclude>com.sematext.spm:spm-monitor-utils</exclude>
                    <exclude>com.sematext.spm:spm-tracing-agent</exclude>
                    <exclude>com.sematext.spm:spm-monitor-starter</exclude>
```